client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/index.tsx

```diff
@@ -46,8 +46,9 @@ const designSetup: Step = function DesignSetup( { navigation } ) {
 	const { setSelectedDesign, setPendingAction } = useDispatch( ONBOARD_STORE );
 	const { setDesignOnSite } = useDispatch( SITE_STORE );
 
-	const anchorPodcastId = getAnchorPodcastId();
-	const flowName = isEnabled( 'signup/anchor-fm' ) && anchorPodcastId ? 'anchor-fm' : 'setup-site';
+	const { anchorFmPodcastId } = useAnchorFmParams();
+	const flowName =
+		isEnabled( 'signup/anchor-fm' ) && anchorFmPodcastId ? 'anchor-fm' : 'setup-site';
```

Comment:
Do we not have a way to get this from the stepper framework? This seems like something that should be available to steps within the flow.

Reply:
Hmm, that's a good point, I will check!

Reply:
The only place I saw the flow name (declared in the flow file) being used is here:
`client/landing/stepper/declarative-flow/internals/index.tsx` 
```
<div className={ classnames( flow.name, flow.classnames, pathToClass( path ) ) }>
```

Maybe they don't want steps to change behavior depending on which flow they are in?

Reply:
Could we set a value for the flow name in the onboarding store when we pass it to `FlowRenderer`?

Reply:
That makes sense, if it's not available already. (/cc @alshakero in case you have any specific opinion on this :) )

Reply:
I think the simplest solution would be to pass the flowname to the [step here](https://github.com/Automattic/wp-calypso/blob/update/anchor-podcast-title-from-api/client/landing/stepper/declarative-flow/internals/index.tsx#L47). 

Like so 
```js
<StepComponent navigation={ stepNavigation } flowName={flow.name} />
```